### ISSUE TITLE
Packaging broken for 0.9.26.20git?

### ISSUE THREAD

**Issue by scott8035** [rncbc/qtractor#335]
I know enough about apt to know better than to proceed with this install, but not enough to know how to fix it... I get the following output when I try to resolve a "The following packages have been kept back" error via "apt-get install qjackctl qtractor". It seems like it wants to downgrade the system to 32-bit? Not sure.

**sudo apt-get install qjackctl qtractor**
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  base-files:i386 gcc-10-base:i386 libc6:i386 libcrypt1:i386 libgcc-s1:i386
Suggested packages:
  glibc-doc:i386 locales:i386
Recommended packages:
  libidn2-0:i386
**The following packages will be REMOVED:
  base-files**
The following NEW packages will be installed:
  base-files:i386 gcc-10-base:i386 libc6:i386 libcrypt1:i386 libgcc-s1:i386
The following packages will be upgraded:
  qjackctl qtractor
**WARNING: The following essential packages will be removed.
This should NOT be done unless you know exactly what you are doing!
  base-files**
2 upgraded, 5 newly installed, 1 to remove and 1 not upgraded.
Need to get 5347 kB of archives.
After this operation, 13.4 MB of additional disk space will be used.
You are about to do something potentially harmful.
To continue type in the phrase 'Yes, do as I say!'

**Comment by rncbc** [comment]
try `apt-get remove qtractor qjackctl` etc. and then `apt-get install qtractor qjackctl` again

the old packages wre being probably linked to qt5 and the newer ones are now on qt6 as they should.

hth.

**Comment by scott8035** [comment]
No dice. Same situation, except now I don't have any Q packages installed :(

Also, qsynth requires libfluidsynth3, but that doesn't seem to be available, unless it's in a repo/ppa somewhere I'm not using.

**Comment by rncbc** [comment]
> Also, qsynth requires libfluidsynth3, but that doesn't seem to be available, unless it's in a repo/ppa somewhere I'm not using.

yes, there are this other repo/ppa named "Libraries (????)" which provides libfluidsynth3 alright (re. fluidsynth >= 2.2; your current distro perhaps still having fluidsynth < 2.2)

what is your distro anymay? (`cat /etc/os-release ; uname -a` ?)

**Comment by scott8035** [comment]
> what is your distro anymay? (`cat /etc/os-release ; uname -a` ?)

NAME="Linux Mint"
VERSION="20.3 (Una)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 20.3"
VERSION_ID="20.3"
HOME_URL="https://www.linuxmint.com/"
SUPPORT_URL="https://forums.linuxmint.com/"
BUG_REPORT_URL="http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/"
PRIVACY_POLICY_URL="https://www.linuxmint.com/"
VERSION_CODENAME=una
UBUNTU_CODENAME=focal

Linux BigDog 5.17.0-5.1-liquorix-amd64 #1 ZEN SMP PREEMPT liquorix 5.17-9ubuntu1~focal (2022-04-27) x86_64 x86_64 x86_64 GNU/Linux

I'll setup a VM and try the Libraries PPA out to make sure I'm not going to hose my main install.

**Comment by rncbc** [comment]
you should then use the "[Applications (focal)](https://launchpad.net/~rncbc/+archive/ubuntu/apps-focal)" and "[Libraries (focal)](https://launchpad.net/~rncbc/+archive/ubuntu/libs-focal)" repos, I guess.
